packages/matter.js/src/protocol/interaction/SubscriptionHandler.ts

```diff
@@ -112,38 +115,61 @@ export class SubscriptionHandler {
     }
 
     private registerNewAttributes() {
-        const attributes = this.attributeRequests !== undefined ? this.endpointStructure.getAttributes(this.attributeRequests) : [];
         const newAttributes = new Array<AttributeWithPath>();
+        const attributeErrors = new Array<TypeFromSchema<typeof TlvAttributeStatus>>();
         const formerAttributes = new Set<string>(this.attributeListeners.keys());
 
-        attributes.forEach(({ path, attribute }) => {
-            formerAttributes.delete(attributePathToId(path));
-
-            const existingAttributeListener = this.attributeListeners.get(attributePathToId(path));
-            if (existingAttributeListener !== undefined) {
-                const { attribute: existingAttribute, listener: existingListener } = existingAttributeListener;
-                if (existingAttribute !== attribute) {
-                    if (existingListener !== undefined) {
-                        existingAttribute.removeValueChangeListener(existingListener);
+        if (this.attributeRequests !== undefined) {
+            this.attributeRequests.forEach(path => {
+                const attributes = this.endpointStructure.getAttributes([path]);
+
+                if (attributes.length === 0) {
+                    const { endpointId, clusterId, attributeId } = path;
+                    if (endpointId === undefined || clusterId === undefined || attributeId === undefined) { // Wildcard path: Just leave out values
+                        logger.debug(`Subscription attribute ${this.endpointStructure.resolveAttributeName(path)}: ignore non-existing attribute`);
```

Comment:
Is this the correct error message?

Reply:
In fcat it is no error message it is just logging. When a wildcard path was given and that one did not matched to any attribute then this is no error according to specs,but ignored.

> if the path is NOT a concrete path ...  perform Request Path Expansion and process each expanded existent path as follows:
> * If the path indicates attribute data that is not readable, then the path SHALL be dis­ carded.
> * Else if reading from the attribute in the path requires a privilege that is not granted to access the cluster in the path, then the path SHALL be discarded.

... nothing more ...

> • If no error free existent paths remain, then AttributeRequests are considered empty.